src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -3009,6 +3043,9 @@ ProduceBoundNode:
                                 End If
 
                                 argumentNamesLocationsBuilder.Add(id.GetLocation())
+                            ElseIf argumentNamesBuilder IsNot Nothing Then
+                                argumentNamesBuilder.Add(Nothing)
+                                argumentNamesLocationsBuilder.Add(Nothing)
```

Comment:
>argumentNamesLocationsBuilder.Add(Nothing) [](start = 32, length = 42)

We probably should do similar adjustments for ```Case SyntaxKind.OmittedArgument```. Please ensure to test scenarios with omitted arguments following named arguments, especially when they are followed by another named argument.